process/elasticproc/accounts/accountsProcessor.go

```diff
@@ -128,31 +128,35 @@ func (ap *accountsProcessor) PrepareRegularAccountsMap(timestamp uint64, account
 			TotalBalanceWithStakeNum: balanceAsFloat,
 			Timestamp:                time.Duration(timestamp),
 			ShardID:                  shardID,
-
-			UserName: userAccount.UserAccount.UserName,
-			Owner:    userAccount.UserAccount.CurrentOwner,
 		}
 
-		ap.addDeveloperRewardsInAccountInfo(userAccount.UserAccount.DeveloperRewards, acc)
+		ap.addAdditionalDataInAccount(userAccount.UserAccount.AdditionalData, acc)
 
 		accountsMap[address] = acc
 	}
 
 	return accountsMap
 }
 
-func (ap *accountsProcessor) addDeveloperRewardsInAccountInfo(developerRewards string, account *data.AccountInfo) {
-	if developerRewards == "" {
+func (ap *accountsProcessor) addAdditionalDataInAccount(additionalData *outport.AdditionalAccountData, account *data.AccountInfo) {
+	if additionalData == nil {
+		return
+	}
+
+	account.UserName = additionalData.UserName
+	account.CurrentOwner = additionalData.CurrentOwner
+
+	if additionalData.DeveloperRewards == "" {
```

Comment:
suggestion to extract L149-L160 in a function that deals only with the developers fees.
Reason: if someone will add further processing on other fields after L160, they won't be executed because of the early return on L150

Reply:
done.